### ISSUE TITLE
[explorer/navigator] implement 'More' toolbar item with context menu

### ISSUE THREAD

**Issue by vince-fugnitto** [theia-ide/theia#5951]
**Description**

Based on https://github.com/theia-ide/theia/issues/5946#issuecomment-521630400

Support _"More"_ toolbar item and context menu for the `explorer` widget which will contain helpful methods for the view (ex: `Add Folder to Workspace`)

A similar implementation is present in the `Git` explorer:

<img src="https://user-images.githubusercontent.com/3082655/63095823-84fc2100-bf6c-11e9-8787-e4add3d7dd00.png" />

**Implementation Details**
1. should the toolbar item be always displayed? or should it only be displayed in a single-root workspace?
2. which context menu items should be displayed?

**Comment by akosyakov** [comment]
It should display what it used to display in the context menu before introducing view containers.

**Comment by vince-fugnitto** [comment]
I started implementing the 'more' toolbar but I realized a lot of the commands will need to be updated / duplicated in order to support the feature. They mostly use a `UriAwareCommandHandler` and since there is no longer a root node present (with a URI) the toolbar is not visible until a selection is made in the widget. I will need to adjust or duplicate the commands to default to the root node if possible.

<div align='center'>

![image](https://user-images.githubusercontent.com/40359487/63101084-5a976d00-bf46-11e9-8848-08199b8d96a5.png)


</div>
